check-deps/Makefile

```diff
@@ -23,17 +23,17 @@ CHECK_CFLAGS  = $(CFLAGS)  $(CFLAGS_$@) -Werror
 CHECK_LDFLAGS = $(LDFLAGS) $(LDFLAGS_$@)
 
 # python3.8 added --embed option
```

Comment:
Could you please update this comment too?

Reply:
updated~